src/login_helper.js

```diff
@@ -61,6 +66,347 @@ function filterPastAndUpcomingTrips(travelerTrips) {
+      tripRequestData.push(destinationIdentifier);
+    }
+  }
+  tripRequestData.push(
+    "Destination: " + tripRequestDestination,
+    "Number of Travelers: " + numberOfTravelers,
+    "Trip Start Date: " + tripRequestStartDate,
+    "Trip End Date:" + tripRequestEndDate
+  );
+  return tripRequestData;
+}
+function renderDestinationImage(destinationInfo) {
+  let destinationImage = destinationInfo.image;
+  var img = document.createElement("img");
+  img.className = "destination-image";
+  img.src = `${destinationImage}`;
+  document.querySelectorAll("section")[3].appendChild(img);
+}
+
+function getValuesFromInputs() {
+  let values = [];
+  let tripRequestDestination = document.getElementById("destination").value;
+  let tripRequestStartDate = document.getElementById("start").value;
+  let tripRequestEndDate = document.getElementById("end").value;
+  let numberOfTravelers = document.getElementById("number-of-travs").value;
+  values.push(
+    tripRequestDestination,
+    tripRequestStartDate,
+    tripRequestEndDate,
+    numberOfTravelers
+  );
+  return values;
+}
+
+function calculateTripRequestCosts(destination) {
+  let costs = [];
+  let values = getValuesFromInputs();
+  let numberOfTravelers = values[3];
+  let tripRequestStartDate = values[1];
+  let tripRequestEndDate = values[2];
+  let flightCosts =
+    numberOfTravelers * destination.estimatedFlightCostPerPerson;
+  let lodgingCostsPerDay =
+    numberOfTravelers * destination.estimatedLodgingCostPerDay;
+  let totalLodgingCosts =
+    lodgingCostsPerDay * daysBetween(tripRequestStartDate, tripRequestEndDate);
+  let costBeforeAgentFee = flightCosts + totalLodgingCosts;
+  let agentFee = costBeforeAgentFee * 0.1;
+  let totalTripCost = costBeforeAgentFee + agentFee;
+  costs.push(
+    flightCosts,
+    lodgingCostsPerDay,
+    totalLodgingCosts,
+    costBeforeAgentFee,
+    agentFee,
+    totalTripCost
+  );
+  return costs;
+}
+
+function renderDestinationName(destination) {
+  let destinationName = document.createTextNode(
+    `Trip Destination: ${destination.destination}`
+  );
+  let destinationNameParagraph = document.createElement("paragraph");
+  destinationNameParagraph.appendChild(destinationName);
+  document.querySelectorAll("section")[3].appendChild(destinationNameParagraph);
+}
+
+function renderNumberOfTravelers() {
+  let values = getValuesFromInputs();
+
+  let numOfTravelersNode = document.createTextNode(
+    `Number of Travelers: ${values[3]}`
+  );
+  let numOfTravelersParagraph = document.createElement("paragraph");
+  numOfTravelersParagraph.appendChild(numOfTravelersNode);
+  document.querySelectorAll("section")[3].appendChild(numOfTravelersParagraph);
+}
+
+function renderStartandEndDate(tripRequestStartDate, tripRequestEndDate) {
+  let formattedStartDate = new Date(tripRequestStartDate);
+  formattedStartDate = new Date(
+    formattedStartDate.getTime() +
+    formattedStartDate.getTimezoneOffset() * 60000
+  );
```

Comment:
a helper function to clean up these lines?
let formattedStartDate = formatStartDate(tripRequestStartDate)